draft-ietf-rats-eat.md

```diff
@@ -1463,6 +1453,17 @@ Further, if there is variation within a format that is allowed, the profile shou
 For example, there are variations in the CoSWID format.
 A profile that require the receiver to accept all variations that are allowed to be sent.
 
+### Message Type Identification
+
+EAT defines only a message format that is carried as a field in some other protocol.
+The identification of the message as an EAT is dependent on the carrying protocol and its type system.
+Identification may be implicit, by media type, by CBOR tag number or other.
+
+EAT profiles therefore do not need to specify how an EAT message is identified as that is less a characteristic of EAT and more a characteristic of the carrying protocol.
+
+That said, there is no prohibition against EAT profiles requiring use of CBOR tags, media types, or even against defining new message type identifiers.
+
+Note also, there are CBOR tag requirements imposed by CWT and COSE that must be adhered to.
```

Comment:
I am mostly indifferent to this change (section 3 already makes the points made in the new text). I'd probably replace the last two sentences with: 
```
EAT profiles may require use of CBOR tags, media types or new message type identifiers, provided any requirements of the carrying protocol are satisfied. 
```
CWT and COSE are not the only requirements generators.